daprdocs/content/en/developing-applications/building-blocks/state-management/state-store-ttl.md

```diff
@@ -28,10 +28,12 @@ Refer to the TTL column in the [state store components guide]({{< ref supported-
 
 You can set state TTL in the metadata as part of the state store set request:
 
-{{< tabs Python "HTTP API (Bash)" "HTTP API (PowerShell)">}}
+{{< tabs Python ".NET" Java Go "HTTP API (Bash)" "HTTP API (PowerShell)">}}
```

Comment:
Two questions
1) If this feature in the Javascript SDK? Should this be included here
2) Can we make the order consistent for the languages, not that this matters greatly, but it would be good if these started with .NET For example https://docs.dapr.io/developing-applications/building-blocks/pubsub/howto-publish-subscribe/